vnext/NuGet.Config

```diff
@@ -5,6 +5,6 @@
   </config>
   <packageSources>
     <add key="Nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="react-native-public" value="https://pkgs.dev.azure.com/ms/react-native/_packaging/react-native-public/nuget/v3/index.json" />
```

Comment:
I'm not sure if external folks will be able to access this? I believe all nuget sources must be reachable.
Can this feed be added only to the desktop project?

Reply:
According to @dannyvv, this feed is explicitly made for public access (hence its name in the URL).
There is no way to make it "Desktop-only", to my knowledge.

Because packages consumed by MSRN are not on this feed, it will be transparent for MSRN users.